products/securitycenter/terraform.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+legacy_name: scc
```

Comment:
This renaming matches the service name used throughout the product documentation ("Cloud SCC") and the `gcloud` command `gcloud scc`.

I think it'll match user expectations better than `security_center`, but if you feel otherwise let me know!